packages/shared/src/components/ProfilePicture.tsx

```diff
@@ -103,7 +104,7 @@ function ProfilePictureComponent(
         alt={`${user.username || user.id}'s profile`}
         onError={onError}
         className={classes}
-        loading={isCompanion ? 'eager' : 'lazy'}
+        loading={eager || isCompanion ? 'eager' : 'lazy'}
```

Comment:
We actually prefer/wanted to move all to `nativeLazyLoading` would it not work on this one?

Reply:
Wdym? It's still native lazy loading, but I wanted to add an option to eagerly load them. The default is lazy load but to optimize the profile page, I wanted above the fold images to load immediately

Reply:
We try to avoid the `lazy` one as it uses js lazy loading (from top of my head) we have a option for `nativeLazyLoading` which uses plain `img` elements props.

Reply:
I believe this is exactly what we do here

Reply:
Oh sigh, I for some reason thought this was the bottom evaluation 🤦